dynatrace/src/main/java/com/dynatrace/opentelemetry/metric/Serializer.java

```diff
@@ -115,14 +124,20 @@ static DimensionList fromLabels(Labels labels) {
     for (DoublePointData point : metric.getDoubleSumData().getPoints()) {
       try {
         Metric.Builder builder = createMetricBuilder(metric, point);
-
         if (isDelta) {
           builder.setDoubleCounterValueDelta(point.getValue());
+          lines.add(builder.serialize());
         } else {
-          builder.setDoubleCounterValueTotal(point.getValue());
+          Double delta = deltaConverter.convertDoubleTotalToDelta(metric.getName(), point);
+          if (delta != null) {
+            builder.setDoubleCounterValueDelta(delta);
+            lines.add(builder.serialize());
+          } else {
+            logger.finest(
+                "Skipping delta line creation, since the value was not present in the cache before this value.");
```

Comment:
Same as above (would make sense to use a constant for the template here as well).